lighthouse-core/report/report-generator.js

```diff
@@ -56,7 +56,7 @@ class ReportGenerator {
 
     // Converts a name to a link.
     Handlebars.registerHelper('nameToLink', name => {
-      return name.toLowerCase().replace(/\s/, '-');
+      return name.toLowerCase().replace(/\s/g, '-');
```

Comment:
could also make the helpers static (and private) functions so they're testable on their own. It would make the tests a little easier to write and run a little faster.